coyote/views.py

```diff
@@ -87,4 +92,7 @@ def home(request):
 
 
 def codeplay(request):
+    """
+    Given request return HttpResponse with that rendered text
```

Comment:
Please take some time to get an understanding of Django.
The function does not "get request" per say. 

The function re-renders the htmlviewer  / codeplay page when this function is called. Please convert to an appropriate format.